### PR TITLE
docs: update main readme

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +64,74 @@ For a list of all configuration options for sumologicextension please refer to
...
+One can find all the available tags pushed to this repository at
+https://gallery.ecr.aws/sumologic/sumologic-otel-collector.
+
+In order to run Sumo Logic OT distro in a container one can use the following
+set of commands:
+
+```bash
+$ ls config.yaml
+config.yaml
+$ docker run --rm -ti --name sumologic-otel-collector -v $(pwd):/etc/ public.ecr.aws/sumologic/sumologic-otel-collector:0.0.4 --config /etc/config.yaml
```
Comment (sumo-drosiek) [inline README.md:117]: Could we explain why we mount full directory?

Comment (pmalek-sumo) [reply README.md]: I didn't think of mounting a single file, thanks! Done.

File: .markdownlint/style.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "MD030": {
+    "ul_multi": 3,
+    "ol_multi": 2
+  },
+  "MD033": false,
+  "MD035": {
+    "style": "---"
+  }
+}
```
Comment (astencel-sumo) [inline .markdownlint/style.json:25]: Missing newline 😱

**Review by astencel-sumo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sumo-drosiek) [reply README.md]: Buuuut, what about state files in the future? :D

File: .markdownlint/style.json (hunk 2 above)
Comment (pmalek-sumo) [reply .markdownlint/style.json]: Thanks, done

**Review by sumo-drosiek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pmalek-sumo) [reply README.md]: Let's cover that in a separate PR :)

**Pull request merged**